file/replica_client.go

```diff
@@ -146,7 +146,7 @@ func (c *ReplicaClient) DeleteGeneration(ctx context.Context, generation string)
 func (c *ReplicaClient) Snapshots(ctx context.Context, generation string) (litestream.SnapshotIterator, error) {
 	dir, err := c.SnapshotsDir(generation)
 	if err != nil {
-		return nil, fmt.Errorf("cannot determine snapshots path: %w", err)
+		return nil, litestream.ErrSnapshotPathNoGeneration
```

Comment:
Why is this removing the error wrapping? The returned `err` from `SnapshotsDir()` could be anything and not just `"no generation"`.

Reply:
I got over zealous with the moving to a general set of errors, fixed.